libs/linking-tool/src/lib/runtime/resources/models/link-params.interface.ts

```diff
@@ -0,0 +1,9 @@
+import { Params } from '@angular/router';
+
+export interface LinkParams {
+  anchor?: string;
+  embedded?: { id: string };
+  queryParams?: Params;
+  skipLocationChange?: boolean;
+  [key: string]: any;
```

Comment:
I am not sure, it's an old code from another project. I think we can remove this for now, if some property will be missing, the change has to be done in this library.

Reply:
I'm not sure we can simply remove this. I believe it is used to interpolate url params (like "somePath/:id/update") e.g. search for `utils.interpolate(url, params)`. What I mean is to move this params under some prop, e.g.
`
skipLocationChange?: boolean;
params?: {  [key: string]: any; }
`
and then pass only `obj.params` instead of  `obj` to the interpolation function. Is it more clear now?